lib/engine/game/g_1860.rb

```diff
@@ -269,6 +309,33 @@ def close_other_companies!(company)
         @log << '-- Event: starting private companies close --'
       end
 
+      def train_help(trains)
+        help = []
+
+        if trains.select { |t| t.owner == @depot }.any?
+          help << 'Leased trains ignore town/halt allowance.'
+          help << "Revenue = #{format_currency(40)} + number_of_stops * #{format_currency(20)}"
+        end
+
+        help
+      end
+
+      def train_owner(train)
+        train.owner == @depot ? lesee : train.owner
+      end
+
+      def lesee
```

Comment:
lesee?

Reply:
Should be "lessee" - someone who is a tenant of a lease. Spelling isn't my strong point.